golang/lib/bulk_command_execution/v0_bulk_command_api/v0_bulk_commands.go

```diff
@@ -137,6 +146,16 @@ func (visitor *cmdArgDeserializingVisitor) VisitExecuteBulkCommands() error {
 	return nil
 }
 
+func (visitor *cmdArgDeserializingVisitor) VisitGetServices() error {
+	visitor.deserializedCommandArgsPtr = nil
```

Comment:
I think this, and the one below, should technically be `protobuf.Empty` (or whatever it's called) rather than `nil`, which will prevent NPEs. That said, someone shouldn't ever be using `GetServices` or `GetLambdas` in bulk command anyways.. :P

Reply:
Yes, it makes sense! it's a better option. Thanks, man! I always appreciate these advices, these help me to improve my knowledge and my code.